src/ui/states/state-present-offer.svelte

```diff
@@ -21,13 +21,13 @@
                 {getTrialsLabel(trial.periodDuration)} free trial
             {/if}
             {#if !trial?.periodDuration && basePrice }
-                {basePrice.currency || ''} {basePrice.formattedPrice}
```

Comment:
I added these changes as well. The `formattedPrice` already includes the currency so it was displaying duplicate in the UI. This removes it so we only display the currency as it comes from the `Intl.NumberFormat` utility.